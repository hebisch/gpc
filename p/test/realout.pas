program testio(input, output);

var
work :array[1..20] of char;
doesit :boolean;
i, j :integer;
r :real;

begin
work := 'This program works? ';
doesit := true;
r := 31.4159;
i := 10;
j := 25;
writeln('i = ',i,' ':3,'j = ':1,j:1);
writeln('r = ',r,'   2 * r = ',2*r);
write('also r = ',r:1,'   and r = ',r:10);
writeln('   finally r = ',r:7:3);
writeln('and r = ',r:j:i);
writeln(work,doesit);
writeln(work:i,doesit:i);
writeln(work:j,doesit:1);
j:=2;
writeln(work:5,doesit:j);
end.
