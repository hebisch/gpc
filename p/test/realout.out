i = 10   j25
r =  3.141590000000000e+01   2 * r =  6.283180000000000e+01
also r =  3.1e+01   and r =  3.142e+01   finally r =  31.416
and r =             31.4159000000
This program works? True
This progr      True
     This program works? T
This Tr
